### PR TITLE
WIP: Workspace scoped steps

### PR DESCRIPTION
**Pull request by catsby**
_Description coming soon_

### CODE REVIEW

File: internal/cli/pipeline_run.go [hunk 1]
```diff
@@ -125,20 +125,33 @@ func (c *PipelineRunCommand) Run(args []string) int {
...
 		successful := steps
 		for _, jobId := range resp.AllJobIds {
-			app.UI.Output("Executing Step %q", resp.JobMap[jobId].Step, terminal.WithHeaderStyle())
+			job, err := c.project.Client().GetJob(c.Ctx, &pb.GetJobRequest{
+				JobId: jobId,
+			})
+			if err != nil {
+				return err
+			}
+			ws := "default-nil"
```
Comment (briancain) [inline internal/cli/pipeline_run.go:134]: Oh should this be `default`?

Comment (catsby) [reply internal/cli/pipeline_run.go]: Good catch Brian, thanks, fixed in 729f4fa9b

**Review by demophoon — APPROVED** [review]
Code looks good to me, thanks for walking us through it

**Review by xiaolin-ninja — APPROVED** [review]
LGTM!

**Pull request merged**